src/setup-k8s/templates/fabric-k8s/scripts/base-functions.sh

```diff
@@ -165,18 +165,25 @@ checkDependencies() {
   if [[ $(command -v kubectl) ]]; then
     printf "\nKubectl installed...\n"
   else
-    printf "\nCouldn't detect a Kubectl \n" && exit
+    printf "\nCouldn't detect Kubectl \n" && exit 1
   fi
 
-  if [[ $(command -v kubectl-hlf) ]]; then
+  if [[ $(export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH" && command -v kubectl-hlf) ]]; then
```

Comment:
I believe we should not update `PATH` by fablo. If this is required in CI, then it should be done in CI yaml when we install k8.

Reply:
fixed in latest commit